galaxy.yml

```diff
@@ -81,7 +81,7 @@ dependencies:
   'git+https://github.com/openstack/ansible-collections-openstack': 'master'
   'git+https://github.com/ansible-collections/ansible.netcommon': 'main'
   'git+https://github.com/openstack/ansible-config_template': 'master'
-  'git+https://github.com/ansible-collections/junipernetworks.junos': '2.10.0'
+  'git+https://github.com/ansible-collections/junipernetworks.junos': 'main'
```

Comment:
Please also update this in the requirements:

https://github.com/openstack-k8s-operators/ci-framework/blob/00bdc9430c01230d6965015274d72af035991cb3/requirements.yml#L39

Reply:
done